source/documentation/deploying-an-app/use-circleci-to-upgrade-app.html.md.erb

```diff
@@ -23,14 +23,78 @@ By the end of the tutorial, you will have done the following:
 It is assumed you have the following:
 
  - You have [created an environment for your application][env-create]
- - You have [deployed an application][deploy-helm] to the 'cloud-platform-live-1' cluster using Helm.
+ - You have deployed the multi-container rails app to the 'cloud-platform-live-1' cluster.
  - You have created an [ECR repository][ecr-setup]
 
 ### Creating a Service Account for CircleCI
 As part of the CircleCI deployment pipeline, CircleCI will need to authenticate with the Kubernetes cluster. In order to do so, Kubernetes uses [Service Accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/). Service Accounts provide an identity for processes that run in a cluster allowing the process to access the API server.
 
 A Service Account is created in the [namespace creation github repository](https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespaces).
 
+If you have not created a Circle CI service account along with the role / rolebinding then you may
+use the below as an example:
+
+```bash
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: circleci
+  namespace: <YOUR-NAMESPACE>
+
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: circleci
+  namespace: <YOUR-NAMESPACE>
+rules:
+  - apiGroups: ["", "extensions", "apps", "batch"]
+    resources:
+      - "pods/portforward"
+      - "deployment"
+      - "secrets"
+      - "services"
+      - "pods"
+      - "jobs"
+    verbs:
+      - "patch"
+      - "get"
+      - "create"
+      - "delete"
+      - "list"
+  - apiGroups:
+      - "extensions"
+    resources:
+      - "deployments"
+      - "ingresses"
+    verbs:
+      - "get"
+      - "update"
+      - "delete"
+      - "create"
+      - "patch"
+
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: circleci
+  namespace: <YOUR-NAMESPACE>
+subjects:
+- kind: ServiceAccount
+  name: circleci
+  namespace: <YOUR-NAMESPACE>
+roleRef:
+  kind: Role
+  name: circleci
+  apiGroup: rbac.authorization.k8s.io
+```
+
+Remember to replace <YOUR-NAMESPACE> with your namespace. Save the above as 'serviceaccount-circleci.yaml' under your [namespace folder](https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespaces).
```

Comment:
You need to escape "<YOUR-NAMESPACE>" with backticks here, or it doesn't show up in the rendered output.

Reply:
done.